frontend/src/components/pages/topics/Topic.List.tsx

```diff
@@ -34,15 +34,16 @@ import {
   useToast,
 } from '@redpanda-data/ui';
 import { AnimatePresence, motion } from 'framer-motion';
-import { autorun, computed, type IReactionDisposer, makeObservable, observable } from 'mobx';
-import { observer } from 'mobx-react';
-import React, { type FC, useRef, useState } from 'react';
+import { useQueryStateWithCallback } from 'hooks/useQueryStateWithCallback';
+import { observable } from 'mobx';
+import { parseAsBoolean, parseAsString, useQueryState } from 'nuqs';
+import React, { type FC, useCallback, useMemo, useRef, useState } from 'react';
 import { HiOutlineTrash } from 'react-icons/hi';
 import { MdError, MdOutlineWarning } from 'react-icons/md';
+import { useCreateTopicMutation, useLegacyListTopicsQuery } from 'react-query/api/topic';
```

Comment:
How much of the fleet is already using console `v3.x`?

Reply:
I like moving to gRPC here, but it's a bit buggy and I think it should be handled in a separate PR.